azurerm/resource_arm_postgresql_server_test.go

```diff
@@ -386,7 +386,7 @@ resource "azurerm_postgresql_server" "test" {
   }
 
   storage_profile {
-    storage_mb = 947200
+    storage_mb = 4194304
```

Comment:
This test will fail. You might want to change the memoryOptimized test https://github.com/gabinante/terraform-provider-azurerm/blob/35a13bb4a1ffff476bd91f99391a31d42e60a48d/azurerm/resource_arm_postgresql_server_test.go#L455